src/components/search/SinopiaSearchResults.jsx

```diff
@@ -1,7 +1,7 @@
 // Copyright 2019 Stanford University see LICENSE for license
 /* eslint max-params: ["error", 4] */
 
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useRef } from 'react'
```

Comment:
Same change is needed for QASearchResults.jsx as well.

Reply:
done, thanks for the help